tests/test_convert.py

```diff
@@ -534,7 +534,7 @@ def test_blocks_to_frames(blocks, block_length):
     assert np.allclose(frames, block_length * np.asanyarray(blocks))
 
     # Check dtype
-    assert np.issubdtype(frames.dtype, np.int)
+    assert np.issubdtype(frames.dtype, int)
```

Comment:
I gather this change has to do with the deprecation warning on using np.int - do we know if checking subtype against python int has equivalent behavior?